etc/kayobe/kolla/globals.yml

```diff
@@ -12,8 +12,8 @@ kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %
 
 kayobe_image_tags:
   openstack:
-    rocky: zed-rocky-9-20230821T155947
-    ubuntu: zed-ubuntu-jammy-20230821T155947
+    rocky: zed-rocky-9-20230921T153510
```

Comment:
I'm a bit confused - the rocky 9 snapshots were only last week and are coming into zed via this open [PR](https://github.com/stackhpc/stackhpc-kayobe-config/pull/692/files)

Reply:
Might've rebuilt them for a different reason. Either way, more up to date containers is more good right?

Reply:
Sure.

Reply:
Especially tested ones